src/custom.scss

```diff
@@ -0,0 +1,7 @@
+// change the theme
+$theme-colors: (
+  'ner-red': #ef4345
+);
+
+// Import Bootstrap and its default variables
+@import '~bootstrap/scss/bootstrap.scss';
```

Comment:
why? why on line 7?

Reply:
You have to modify the variables before importing, or else they'll be loaded without any modification